packages/client/wallets/aa/src/types/API.ts

```diff
@@ -6,14 +6,19 @@ export type StoreAbstractWalletInput = {
     userIdentifier: UserIdentifier;
     type: string;
     smartContractWalletAddress: string;
-    eoaAddress: string;
-    sessionKeySignerAddress: string;
+    signerData: EOASignerData;
```

Comment:
Are we adding the `PasskeysSignerData` in another PR?

Reply:
Yeah coming soon